### DISCUSSION TITLE
Bug in relation HasOneThrough

### DISCUSSION THREAD

**Discussion by doox911** [laravel/framework#33337] [General]
### Russian version

```
$shape = Position::with('shape')->find($position_id)->shape;
$copy_shape = $shape->replicate();
$copy_shape->save();
```

Получаю модель через отношение **HasOneThrough**. Затем, сохраняю копию модели (метод **replicate()**) . А затем сохраняю эту копию, то получаю ошибку:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'laravel_through_key' in 'field list' (SQL: insert into `shapes..` 
```

1. Зачем необходимо св-во  **laravel_through_key**?
2. Почему это свойство не игнорируется при сохранении?

### English version

I get the model through the relation **HasOneThrough**. Then, I save a copy of the model (method **replicate ()**). And when I save this copy, I get an error:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'laravel_through_key' in 'field list' (SQL: insert into `shapes..` 
```

**Laravel version is 7.4.0**

1. Why is  **laravel_through_key** necessary?
2. Why is this property not ignored when saved?

**Comment by jmarcher** [comment]
You could remove the `with('shape')` part or just do `$copy_shape = $shape->fresh()->replicate();` I believe that should work.
